multisig/helpers/config.ts

```diff
@@ -25,5 +26,8 @@ export const ADDRESSES: NetworkAddress = {
     VUSDT: "0x2c3Cc8EED38b4d323ce97DB11f98737F3F7421A1",
     VUSDC: "0x6f1089e90C6a14397C20348eE704d596819DFEd4",
     VTREASURY: "0xD51008eAe81475b542638433a617f5f4171b39ee",
+    XVS: "0x1be95611FC9A808F8794bc9164223b1Fcf49C8Bd",
```

Comment:
I suppose we should use here the "real" XVS on sepolia: 0xdb633c11d3f9e6b8d17ac2c972c9e3b05da59bf9